client/question/question.js

```diff
@@ -639,7 +639,7 @@ tie.constant('RUNTIME_ERROR_FEEDBACK_MESSAGES', {
       return errorString.startsWith('KeyError: ');
     },
     generateMessage: function(errorString) {
-      var keyErrorRegEx = /KeyError:\s(\w+)\s/;
+      var keyErrorRegEx = /KeyError:\s('.*'|\w+)\s/;
```

Comment:
Can we add a more deterministic indicator, like "on line" at the end?

This helps guard against cases where the key itself contains spaces.

Also I'm not sure you need the \w+ part?

Reply:
I don't think the existing test is s a valid test, actually. If you look at the message, it doesn't realy make sense, it seems like it was done that way just for expediency and seems to be several error messages smushed together.

Probably don't need to require quotes in the regex.